.github/workflows/build.yml

```diff
@@ -36,11 +45,16 @@ jobs:
       run: |
         git clone --single-branch --depth 1 -b ${{ inputs.branch }} https://github.com/${{ inputs.fork }}/dotnet
 
+    - name: Apply patches
+      run: |
+        # needed for .NET 10 linux-musl-riscv64, remove it once we move on from .NET 10
+        curl -sSL https://github.com/dotnet/runtime/pull/121662.patch | git -C dotnet/src/runtime apply || true
```

Comment:
The patch applies cleanly to v10. On main, it will fail silently (`|| true`) since the upstream PR is merged and the VMR will sync the changes soon.